waypoints:
  station_a:
    x: -0.650436
    y: -0.571368
    yaw: 0.013600

  station_b:
    x: 0.483058
    y: -0.555954
    yaw: 0.013604

  station_c:
    x: 1.999281
    y: 0.362346
    yaw: 1.587410

  station_d:
    x: -0.013718
    y: 0.584281
    yaw: 3.065955

  docking_station:
    x: -1.667533
    y: 0.709613
    yaw: 3.065957

  home:
    x: -1.999942
    y: -0.500001
    yaw: 0.0
